client_test.go

```diff
@@ -13,7 +13,10 @@ import (
 	"golang.org/x/net/context"
 )
 
-const exampleIssuer = "https://issuer.org"
+const (
+	exampleIssuer = "https://issuer.org"
+	zeroTime = 0 * time.Second
```

Comment:
maybe DefaultKeySetExpiration? Because it's really a meaning of this constant

Reply:
hm, this is only for test